std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
I tend to agree with you. I'm pretty sure std.encoding is *really* old, so we just have to deal with the cruft.

I'd say just slap on the tags, and see whether anyone complains :) If they do, we can figure out a way to accommodate them.